### ISSUE TITLE
Dockerfile / Docker image

### ISSUE THREAD

**Issue by sammcj** [archSeer/colibri-server#1]
would be great to have an official docker image / dockerfile!

**Comment by archseer** [comment]
Indeed! The project is still very much experimental (frontend hardcodes the API at the moment, and so on..), so I haven't figured out packaging yet. I think the general idea would be to merge the two repositories, then for (container) builds, compile the Ember frontend and let it be served with the Elixir server.

Will work on getting it easier to setup and develop this week, in order to encourage contributions :) That said, I have limited time to work on this daily, so I'll probably need a few days. I plan on also sorting out the README and more publicly announcing this once it's a bit more user friendly.
